examples/java/hello_world/BUILD

```diff
@@ -3,10 +3,7 @@ licenses(["notice"])  # Apache 2
 android_binary(
     name = "hello_envoy",
     srcs = glob([
-        "MainActivity.java",
-        "Response.java",
-        "ResponseRecyclerViewAdapter.java",
-        "ResponseViewHolder.java",
+        "*.java",
```

Comment:
I somewhat prefer listing these dependencies explicitly, since it allows more than one module per directory (this is used all over the Envoy project).

Reply:
ok